app/src/components/views/login.jsx

```diff
@@ -63,40 +75,18 @@ export class LoginView extends React.Component {
 
   disableKeyLogger() {
     this.currentRfid = '';
-    $(document).off("keypress");
+    document.removeEventListener('keypress', (...a) => this.handleKeyPress(...a));
```

Comment:
This won't actually remove the correct event listener, if you specify the callback to remove you have to pass a reference to the original callback.

Reply:
Good point. Will extract 'em now and fixy-fixy.

Reply:
There @Aborysa, should be fixed now.